alleyinteractive/searchpress
# Issue-187: Page size is displayed incorrectly

### Summary

This PR addresses the issue described in [Page size is displayed incorrectly](https://github.com/alleyinteractive/searchpress/issues/187). It corrects the page size display when running the command with a page size of 1000 or above.

Fixes #187

### Description

When the page size is set to 1000 or above, the message displayed in the CLI is incorrect. This fix ensures that the correct page size is displayed, improving the accuracy of the feedback provided to the user. The update involves a change to expect a string for the page size, ensuring that larger numbers are handled correctly.

### How to Test

1. Run the command `wp searchpress index --flush --put-mapping` with a page size of 1000 or higher.
2. Observe the output in the CLI to confirm that it correctly states the number of posts being indexed at a time.

### Additional Information

_No additional information provided._
